guide/payments/request.md

```diff
@@ -457,11 +458,11 @@ For in-person payments, showing the confirmation screen to the sender will likel
 
 ---
 
-Let's go, time go dive in a bit deeper with [transaction privacy]({{ "/guide/payments/privacy/" | relative_url }}).
+Next we go over the different types of [payment request formats](/guide/payments/request/payment-request-formats/) available in bitcoin.
```

Comment:
```suggestion
Next, we go over the different [payment request formats](/guide/payments/request/payment-request-formats/) available in bitcoin.
```
The comma is grammatically necessary. Removing "types of" is merely my opinion. "Types of" changes the meaning of the sentence. We are actually going over _different payment request formats_ in the chapter; "types of" implies we are looking at "categories of payment request formats" and not the payment request formats themselves.